src/css_composer/view/CssRulesView.js

```diff
@@ -104,10 +104,7 @@ export default Backbone.View.extend({
 
   getMediaWidth(mediaText) {
     return (
-      mediaText &&
-      mediaText
-        .replace(`(${this.em.getConfig('mediaCondition')}: `, '')
-        .replace(')', '')
+      mediaText && mediaText.replace(/\((min|max)-width: /, '').replace(')', '')
```

Comment:
Well, I know that min/max-width are the most used conditions but what if I want to use something completely custom. I'd leave your new one, but I'd restore the replace with mediaCondition from the config